<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewWindow</class>
 <widget class="QDialog" name="ReviewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>21</y>
     <width>22</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Description">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>81</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Title">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>43</y>
     <width>256</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>103</y>
     <width>256</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mark">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_mark">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Image">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
